feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```diff
@@ -843,10 +892,23 @@ func TestImportExportMultifacetMatchActionsBGPPolicy(t *testing.T) {
 	configureFlowV6(t, bs)
 
 	bs.PushAndStartATE(t)
+	time.Sleep(time.Second * 10)
```

Comment:
This should be a gnmi.Await call for the desired number of packets with a timeout:

You can use this helper function:

https://github.com/openconfig/featureprofiles/blob/d63d627e48c1586ea2dc613500b8ecbc18dc2e8a/internal/otgutils/actions.go#L27